pkg/server/user.go

```diff
@@ -547,7 +547,7 @@ func (s *ImmuServer) getValidatedUser(username []byte, password []byte) (*auth.U
 }
 
 // getUser returns userdata (username,hashed password, permission, active) from username
-func (s *ImmuServer) getUser(username []byte, includeDeactivated bool) (*auth.User, error) {
```

Comment:
was not used to reactivate a user?

Reply:
All places in the code call this function with the `true` value. There's only a call in a unit test that uses `false` here.

I decided to remove it though for another reason. The code is actually completely broken since the condition in line 568 should be `if !usr.Active` (Which I found while testing sysadmin password reset). So since it's not used, broken and incorrectly tested, removing it seemed like a logical thing to do.